DEVGUIDE.md

```diff
@@ -116,10 +116,10 @@ Note: if you face this error [#2351](https://github.com/Microsoft/visualfsharp/i
 
 >  error VSSDK1077: Unable to locate the extensions directory. "ExternalSettingsManager::GetScopePaths failed to initialize PkgDefManager for C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\Common7\IDE\devenv.exe".
 
-delete those folders:
+Or hard crash on launch ("Unknown Error"), delete these folders:
 
-* %appdata%\Microsoft\VisualStudio\15.0_<some number here>FSharpDev
-* %appdata%\..\Local\Microsoft\VisualStudio\15.0_<some number here>FSharpDev
+* `%appdata%\Microsoft\VisualStudio\15.0_(some number here)FSharpDev`
+* `%localappdata%\..\Local\Microsoft\VisualStudio\15.0_(some number here)FSharpDev`
```

Comment:
No ... the listed folders should be the right ones.
This is actually incorrect:
%appdata%\Microsoft\VisualStudio\15.0_(some number here)FSharpDev

And this one has a redundant ..\Local  : 
%localappdata%\..\Local\Microsoft\VisualStudio\15.0_(some number here)FSharpDev

and I don't think the ticks add anything except making copy and paste trickier :-)